iree_tests/pytorch/models/sdxl-scheduled-unet-3-tank/real_weights_data_flags.txt

```diff
@@ -0,0 +1,8 @@
+--parameters=model=real_weights.irpa
+--module=sdxl_scheduled_unet_pipeline_fp16_.vmfb
+--input=1x4x128x128xf16=@inference_input.0.bin
+--input=2x64x2048xf16=@inference_input.1.bin
+--input=2x1280xf16=@inference_input.2.bin
+--input=1xf16=@inference_input.3.bin
+--expected_output=1x4x128x128xf16=@inference_output.0.bin
+--expected_f16_threshold=1.5f
```

Comment:
Is `1.5f` a large or small threshold for f16? I'm not really sure with these programs...

Reply:
Dropped it down to 0.8, still pretty high. There are some cpu numerics that Ean is looking into. Once he figures it out and merges the sdxl branch in turbine, we can update any flags if needed and tighten the accuracy threshold.